From b2fd1c0211faba60c7f65550fcefc143583b6171 Mon Sep 17 00:00:00 2001
From: Yannick MOREL <yannickx.morel@intel.com>
Date: Wed, 29 May 2013 12:40:55 +0200
Subject: [PORT FROM MAIN] bionic: update gsmmux.h

BZ: 111558

Update gsmmux.h with missing GSMIOC_DEMUX ioctl definition

Change-Id: Ic94611886159360ebee89372c184854aaaf59ceb
Orig-Change-Id: I044669b1302736c09fbc6cbfb8ed4cd2335118cd
Signed-off-by: Yannick MOREL <yannickx.morel@intel.com>
Reviewed-on: http://android.intel.com:8080/110480
Reviewed-by: cactus <cactus@intel.com>
Reviewed-by: Cohen, David A <david.a.cohen@intel.com>
Reviewed-by: Yang, Fei <fei.yang@intel.com>
Reviewed-by: Chen, HuaqiangX <huaqiangx.chen@intel.com>
Reviewed-by: Yao, Yong <yong.yao@intel.com>
Reviewed-by: Akue, LoicX <loicx.akue@intel.com>
Reviewed-by: Ravindran, Arun <arun.ravindran@intel.com>
Reviewed-by: Lebsir, SamiX <samix.lebsir@intel.com>
Tested-by: Lebsir, SamiX <samix.lebsir@intel.com>
Reviewed-by: buildbot <buildbot@intel.com>
Tested-by: buildbot <buildbot@intel.com>
---
 libc/kernel/common/linux/gsmmux.h |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/libc/kernel/common/linux/gsmmux.h b/libc/kernel/common/linux/gsmmux.h
index 6b06a42..fc2ac8e 100644
--- a/libc/kernel/common/linux/gsmmux.h
+++ b/libc/kernel/common/linux/gsmmux.h
@@ -42,6 +42,7 @@ struct gsm_config {
 #define GSMIOC_GETCONF _IOR('G', 0, struct gsm_config)
 /* WARNING: DO NOT EDIT, AUTO-GENERATED CODE - SEE TOP FOR INSTRUCTIONS */
 #define GSMIOC_SETCONF _IOW('G', 1, struct gsm_config)
+#define GSMIOC_DEMUX _IO('G', 4)
 struct gsm_netconfig {
  unsigned int adaption;
  unsigned short protocol;
-- 
1.7.4.1

